docs/design/metrics.md

```diff
@@ -1,39 +1,40 @@
+|---------------------------------------------|-----------------|------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------|
+| `e2e_scheduling_latency_milliseconds`       | Histogram       | None                                                                                     | End-to-end scheduling latency in milliseconds (scheduling algorithm + binding) |
+| `e2e_job_scheduling_latency_milliseconds`   | Histogram       | None                                                                                     | End-to-end job scheduling latency in milliseconds                              |
+| `e2e_job_scheduling_duration`               | Gauge           | `job_name`=&lt;job_name&gt;, `queue`=&lt;queue&gt;, `job_namespace`=&lt;job_namespace&gt;| End-to-end job scheduling duration                                             |
+| `e2e_job_scheduling_start_time`             | Gauge           | `job_name`=&lt;job_name&gt;, `queue`=&lt;queue&gt;, `job_namespace`=&lt;job_namespace&gt;| End-to-end job scheduling start time                                           |
+| `plugin_scheduling_latency_milliseconds`    | Histogram       | `plugin`=&lt;plugin_name&gt;, `OnSession`=&lt;OnSession&gt;                              | Plugin scheduling latency in milliseconds                                      |
+| `action_scheduling_latency_milliseconds`    | Histogram       | `action`=&lt;action_name&gt;                                                             | Action scheduling latency in milliseconds                                      |
+| `task_scheduling_latency_milliseconds`      | Histogram       | None                                                                                     | Task scheduling latency in milliseconds                                        |
 
 
-### kube-batch operations
-This metrics describe internal state of kube-batch.
+### volcano operations
+This metrics describe internal state of volcano.
 
-| Metric name | Metric type | Labels | Description |
-| ----------- | ----------- | ------ | ----------- |
-| pod_schedule_errors | Counter |  | The number of kube-batch failed due to an error |
-| pod_schedule_successes | Counter | | The number of kube-batch success in scheduling a job |
-| pod_preemption_victims | Counter | | Number of selected preemption victims |
-| total_preemption_attempts | Counter |  | Total preemption attempts in the cluster till now |
-| unschedule_task_count | Counter | `job`=&lt;job_id&gt; | The number of tasks failed to schedule |
-| unschedule_job_counts | Counter | | The number of job failed to schedule in each iteration |
-| job_retry_counts | Counter | `job`=&lt;job_id&gt; | The number of retry times of one job |
+| **Metric Name**               | **Metric Type** | **Labels**                       | **Description**                                      |
```

Comment:
Are `pod_schedule_errors` and `pod_schedule_successes` deleted?

Reply:
I see, I've not found the metrics, like `pod_schedule_errors ` and `pod_schedule_successes `.

Reply:
schedule_attempts_total. Maybe, volcano uses the `schedule_attempts_total` counter metrics.